Dockerfile.windows

```diff
@@ -1,27 +1,63 @@
 FROM alpine AS build
```

Comment:
This should be pinned, for example `alpine:3.12` as used by the golang image in the other Dockerfiles

Reply:
Done, pinned `alpine:3.14`